docs/src/reference/remote_inputs.rst

```diff
@@ -54,6 +54,9 @@ Each regional build (``global``, ``africa``, ``asia``, ``europe``, ``north-ameri
 
    Please note that these files are uploaded in two batches (see above for details). This means that the full GenBank metadata and sequences are typically updated a couple of hours before the more processed files.
 
+.. warning::
+  The zstandard (zstd) files are not yet supported as direct inputs for the pipeline.
```

Comment:
Hmm, looks like they didn't do a release post-merge yet...

We should probably still add a note to users that they need to use a the specific version of `xopen`. Since the ncov sanitize scripts are using `augur.io`, we should probably update the pin of `xopen` version in Augur and do an Augur release. Then we can update the docs here to say users must be using the Augur version XX.X.X to use zst inputs.

Reply:
I created https://github.com/nextstrain/augur/issues/1073 we can discuss specifics there.